client/landing/stepper/declarative-flow/internals/index.tsx

```diff
@@ -62,9 +64,12 @@ export const FlowRenderer: React.FC< { flow: Flow } > = ( { flow } ) => {
 	);
 
 	const urlQueryParams = useQuery();
+	const siteSlugOrIdInUrl = urlQueryParams.get( 'siteSlug' ) || urlQueryParams.get( 'siteId' );
```

Comment:
[nit] We can simply use `useSiteData` hook 😆

Reply:
TIL!